<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dspTax1099</class>
 <widget class="QWidget" name="dspTax1099">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info for 1099's</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>From Date:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="DLineEdit" name="_fromdate"/>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="_sum1099">
           <property name="text">
            <string>Summary 1099</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QRadioButton" name="_detail1099">
           <property name="text">
            <string>Detail 1099</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>To Date:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="DLineEdit" name="_todate"/>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="_sumall">
           <property name="text">
            <string>Summary All</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QRadioButton" name="_detailall">
           <property name="text">
            <string>Detail All</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QWidget" name="widget_2" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="_listLit">
         <property name="text">
          <string>Vendor Payment Information</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="XTreeWidget" name="_list">
         <property name="populateLinear">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="_close">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_query">
         <property name="text">
          <string>Query</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_print">
         <property name="text">
          <string>Print</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="_p1099">
         <property name="text">
          <string>Print 1099</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_p1096">
         <property name="text">
          <string>Print 1096</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="_setaccnt">
         <property name="text">
          <string>Select Accounts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_chg1099">
         <property name="text">
          <string>Select Vendors</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DLineEdit</class>
   <extends>QWidget</extends>
   <header>datecluster.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_fromdate</tabstop>
  <tabstop>_todate</tabstop>
  <tabstop>_sum1099</tabstop>
  <tabstop>_sumall</tabstop>
  <tabstop>_detail1099</tabstop>
  <tabstop>_detailall</tabstop>
  <tabstop>_list</tabstop>
  <tabstop>_close</tabstop>
  <tabstop>_query</tabstop>
  <tabstop>_print</tabstop>
  <tabstop>_p1099</tabstop>
  <tabstop>_p1096</tabstop>
  <tabstop>_setaccnt</tabstop>
  <tabstop>_chg1099</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
